components/card/README.md

```diff
@@ -292,17 +292,22 @@ The `d2l-card-footer-link` element is an icon link that can be placed in the `fo
 | Property | Type | Description |
 |--|--|--|
 | `icon` | String, required | Preset icon key (ex. "tier1:gear") |
-| `text` | String, required | Accessible text for the link (not visible, gets announced when user focuses) |
 | `download` | Boolean | Download a URL instead of navigating to it |
 | `href` | String | Location for the primary action/navigation |
 | `rel` | String | Relationship of the target object to the link object |
 | `secondary-count` | Number | Count to display as a superscript on the icon |
 | `secondary-count-type` | String | Controls the style of the secondary count bubble; options are `notification` and `count` |
 | `secondary-count-max-digits` | Number | Overrides the default maximum digits of the secondary count bubble (2 for `notification` and no limit for `count` type) |
 | `target` | String | Where to display the linked URL |
+| `text` | String | Accessible text for the link |
 
 See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
+<!-- docs: end hidden content -->
+
+## Accessibility
+- The `text` property for the d2l-card and d2l-card-footer-link components is only required if actionable (i.e. the `href` property is used)
```

Comment:
Since this is only true for the `d2l-card` it could probably go in `text` description for the card

Reply:
Makes sense! I've moved it back 👍